Gemfile

```diff
@@ -53,3 +53,12 @@ gem 'faker', group: %i[development test]
 gem 'rspec-rails', group: %i[development test]
 gem 'shoulda-matchers', require: false, group: %i[development test]
 gem 'simplecov', require: false, group: %i[development test]
+
+# Automated code review
+gem 'pronto', group: %i[development test]
+gem 'pronto-brakeman', require: false, group: %i[development test]
+gem 'pronto-eslint', require: false, group: %i[development test]
+gem 'pronto-flay', require: false, group: %i[development test]
+gem 'pronto-rails_best_practices', require: false, group: %i[development test]
+gem 'pronto-rubocop', require: false, group: %i[development test]
+gem 'codeclimate-test-reporter', group: %i[test]
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `codeclimate-test-reporter` should appear before `pronto-rubocop`.